target.json

```diff
@@ -2,7 +2,7 @@
   "name": "nrf51dk-armcc",
   "version": "0.0.8",
   "inherits": {
-    "nordic-nrf51822-32k-armcc": "*"
+    "nordic-nrf51822-armcc": "*"
```

Comment:
Ok, I will make the change. However, "^0.0.0" does not work. I get the following error:
```
error: could not install target nordic-nrf51822-gcc at ^0.0.0 for mkit-gcc
error: The targets registry does not provide a version of "nordic-nrf51822-gcc" matching "==0.0.0"
error: NOTE: use "yotta link-target" to test a locally modified target prior to publishing.
```
When `nordic-nrf51822-gcc 0.0.1`. Nevertheless "^0.0.1" works. Is this acceptable?